Cargo.toml

```diff
@@ -9,14 +9,43 @@ repository = "https://github.com/gfx-rs/naga"
 keywords = ["shader", "SPIR-V"]
 license = "MPL-2.0"
 
+[lib]
+crate-type = ["cdylib", "rlib"]
```

Comment:
Unfortunately that would make the `cdylib` target to be generated for *every* build of naga as a dependency, even of other Rust code. See https://github.com/rust-lang/cargo/issues/4280 .

Reply:
Hmmm, not sure how to solve that, wasm-pack told me this is needed for being able to compile to wasm 😟